src/graphic/helper/roundSector.ts

```diff
@@ -138,38 +174,42 @@ export function buildPath(ctx: CanvasRenderingContext2D | PathProxy, shape: {
         arc = mathAbs(tmpAngles[0] - tmpAngles[1]);
     }
 
-    const cx = shape.cx;
-    const cy = shape.cy;
-    const cornerRadius = shape.cornerRadius || 0;
-    const innerCornerRadius = shape.innerCornerRadius || 0;
-
     // is a point
     if (!(radius > e)) {
         ctx.moveTo(cx, cy);
     }
     // is a circle or annulus
     else if (arc > PI2 - e) {
-        ctx.moveTo(
-            cx + radius * mathCos(startAngle),
-            cy + radius * mathSin(startAngle)
-        );
+        const { x, y } = calcCircleCenter(cx, cy, radius, startAngle);
+        ctx.moveTo(x, y);
         ctx.arc(cx, cy, radius, startAngle, endAngle, !clockwise);
 
         if (innerRadius > e) {
-            ctx.moveTo(
-                cx + innerRadius * mathCos(endAngle),
-                cy + innerRadius * mathSin(endAngle)
-            );
+            const { x, y } = calcCircleCenter(cx, cy, innerRadius, endAngle);
```

Comment:
I will prefer the original code here. From my side it's pretty intuitive already. Using function is worse on the performance because there is an extra call and will create a temporary object. Also the compiled code will be longer:
```js
// Compiled result.
var a=b(c,d,e,f),x=a.x,y=a.y;ctx.moveTo(x,y);
// Previous
ctx.moveTo(a+b*c(d),a+b*c(d));
```

Reply:
Good catch! It has been recoverd.